Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -67,6 +68,12 @@ void InitSoundStream()
     g_sound_stream->Init();
   }
 
+  // Ideally these two calls would be done in AudioInterface::Init so that we don't
+  // need to have a dependency on AudioInterface here, but this has to be done
+  // after creating g_sound_stream (above) and starting audio dumping (below)
```

Comment:
One thought is that `InitSoundStream` could take the rates?  And then `HW` could expose them via getters and those values could be passed in `Core.cpp`.  Not sure if that's worth it though?

Reply:
Yeah, it would work but I'm not sure if it's worth it. Does anyone else have an opinion on this?